_dprhtml/js/dprviewmodel.js

```diff
@@ -173,8 +173,8 @@ const dprCommandList = [
     execute: emptyFn,
     visible: false,
     isDynamic: true,
-    title: "Open relative section in Mūla side by side (Keyboard shortcut: m). Shift+click to open in same pane.",
-    matchKey: e => e.key === 'm',
+    title: "Open relative section in Mūla side by side (Keyboard shortcut: m). Shift+click to open in same pane (Keyboard shortcut: M).",
+    matchKey: e => e.key === 'm' || e.key === 'M',

```

Comment:
I didn't want to get too clever here -- I think it's better to be explicit than to do something like:
```typescript
matchKey: e => e.key.toLowerCase() === 'm',
```

The same goes for the `A` and `T` shortcuts as well.